src/contentscript/overlay/root/components/OverlayTab/index.tsx

```diff
@@ -141,6 +160,14 @@ export const OverlayTab = (p: OverlayTabProps): ReactElement => {
     const url = `${DAPPLETS_STORE_URL}/#searchQuery=${f}`
     window.open(url, '_blank')
   }
+  const getNotifications = async () => {
+    const backgroundFunctions = await initBGFunctions(browser)
+    const { getNotifications, setRead } = backgroundFunctions
+
+    const notifications = await getNotifications(NotificationType.Application)
+
+    return notifications
+  }
   return (
```

Comment:
line break please

Reply:
+
I use linters' scripts and lines deleted